packages/react-scripts/template/README.md

```diff
@@ -1066,6 +1066,17 @@ To override this, specify the `homepage` in your `package.json`, for example:
 
 This will let Create React App correctly infer the root path to use in the generated HTML file.
 
+#### Setting Homepage to "."
```

Comment:
I'd rather go with paths because you don't need to rebuild to serve from `mysite1.com` and `mysite2.com`, only for different paths.